### DISCUSSION TITLE
NameError: name 'Path' is not defined

### DISCUSSION THREAD

**Discussion by Qertile** [nasa/fprime#1600] [General]
Hi everyone,

I met an issue while I was tring to follow the step "Testing F´ GDS Installation Via Running HTML GUI" with [Installation Guide](https://nasa.github.io/fprime/INSTALL.html#Troubleshooting)
The NameError happened at run_deployment.py as I run the command
` fprime-gds -g html -r <path to fprime checkout>/Ref/build-artifacts `

Then I found it seems missed` from pathlib import Path ` at the begining of run_deployment.py

After that a webpage titled as "Sending Command: cmdDisp.CMD_NO_OP" successfully shows up.

Is this error occured due to any inappropriate installation in previous steps?

Thanks!

Following is my environment:
- Linux: Ubuntu LTS 20.04
- F Prime: latest / branch: devel
- Python: 3.9.11
- CMake: 3.24.0
- OpenSSL: 1.1.1f

**Comment by ThibFrgsGmz** [comment]
Look [here](https://github.com/fprime-community/fprime-gds/issues/86), I think @rmelick-muon has fixed this bug in the latest version of fprime-gds: [v3.1.2](https://github.com/fprime-community/fprime-gds/releases/tag/v3.1.2)

Do you have the latest version?

**Reply by Qertile** [reply]
I didn't notice the latest version v3.1.2, I just followed the installation guide. So I checked my fprime-gds version, is v3.1.1. 

Now is up to date, thank you so much!
